src/components/label/label.stories.ts

```diff
@@ -77,6 +77,10 @@ export const simple = (): string => html`
         <calcite-option>c</calcite-option>
       </calcite-select>
     </calcite-label>
+    <calcite-label>
+      Add Notes
+      <calcite-text-area> </calcite-text-area>
```

Comment:
Can you double check all `calcite-text-area` usages don't have a space?

This applies to other files as well (e.g., stories).